src/service/poc_lora.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+enum data_rate {
+  SF7BW125 = 0;
+  SF8BW125 = 1;
+  SF9BW125 = 2;
+  SF10BW125 = 3;
+  SF12BW125 = 4;
+  // TODO: add final values
+}
+
+enum invalid_reason {
+  bad_signature = 0;
+  // TODO: add final values
+}
+message beacon_report_req_v1 {
+  bytes beacon_id = 1;  // likely be a concatenation of local and remote entropy so maybe not needed if we already include both fields ? 
```

Comment:
this is a good question as we'd need to verify it was derived correctly anyway

Reply:
I think it's ok to leave it here though

Reply:
I'd remove this to avoid duplication?

Reply:
removed here: cc7a2e1644eeeae1d54d94e70e35611f3ea459e5